sys-libs/libxcrypt/libxcrypt-4.4.28.ebuild

```
@@ -177,6 +216,12 @@ src_install() {
 	# Remove useless stuff from installation
 	find "${ED}"/usr/share/doc/${PF} -type l -delete || die
 	find "${ED}" -name '*.la' -delete || die
+
+	# workaround broken upstream cross-* --docdir by installing files in proper locations
+	if is_cross; then
+		install -d "${ED}$(get_xcprefix)"/usr/share
```

In line with other bits:
```suggestion
		install -d "${D}$(get_xcprefix)"/usr/share || die
```

... but please do `insinto` instead (and `doins`)